packages/docs/tailwind.config.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const config = require('@dailydotdev/shared/tailwind.config');
+
+module.exports = {
+  ...config,
+  content: [
+    './src/**/*.{js,ts,jsx,tsx}',
+    './node_modules/@dailydotdev/shared/src/**/*.{js,ts,jsx,tsx}',
```

Comment:
We don't need `.js` files?

Reply:
Not really, it's just the default config we use accross the board, so thought best to keep it?

Reply:
From what I can see we don't have any JS files in shared with tailwind classes usages so I don't think its needed https://tailwindcss.com/docs/configuration#content